operate/webapp/src/main/java/io/camunda/operate/webapp/elasticsearch/ElasticsearchSessionRepository.java

```diff
@@ -168,28 +183,36 @@ private Optional<OperateSession> documentToSession(Map<String, Object> document)
                         name, deserialize(Base64.getDecoder().decode(attributes.get(name)))));
       }
       return Optional.of(session);
-    } catch (Exception e) {
+    } catch (final Exception e) {
       LOGGER.error("Could not restore session.", e);
       return Optional.empty();
     }
   }
 
-  private Instant getInstantFor(final Object object) {
-    if (object == null) {
-      return null;
+  private void setPollingFor(final OperateSession session) {
+    try {
+      if (request != null && request.getHeader(POLLING_HEADER) != null) {
+        LOGGER.info("Set session polling to true");
+        session.setPolling(true);
+      }
+    } catch (final Exception e) {
+      LOGGER.debug(
+          "Expected Exception: is not possible to access request as currently this is not on a request context");
     }
-    if (object instanceof Long) {
-      return Instant.ofEpochMilli((Long) object);
+  }
+
+  private Instant getInstantFor(final Object object) {
+    final var instantAsLong = (Long) object;
```

Comment:
This previous had an instanceof check, do we need to keep that to ensure we don't get an exception when casting?